conf/messages

```diff
@@ -13,8 +13,6 @@ common.radioOption.no = No
 
 app.common.back = Back
 
-error.summary.title = There is a problem
```

Comment:
This is being used in summary.scala.html and errorSummary.scala.html

Reply:
Was a duplicate text. Its still in the document on line 569

Reply:
😮